examples/regression/1-quickstart/plot_regression_housing_dataset.py

```diff
@@ -0,0 +1,426 @@
+
+warnings.filterwarnings("ignore")
+
+##############################################################################
+# 1. Data
+# --------------------------------------------------------------------------
+# The target variable is the median house value for California districts. As
+# the value is expressed in thousands of dollars we will times it by 100 to
+# view the dataset in thousands.
+
+
+data = fetch_california_housing(as_frame=True)
+X = pd.DataFrame(data=data.data, columns=data.feature_names)
+y = pd.DataFrame(data=data.target)*100
+
+##############################################################################
+# Let's visualize the two-dimensional dataset, the correlations between the
+# independent variables.
+
+
+df = pd.concat([X, y], axis=1)
+pear_corr = df.corr(method='pearson')
+pear_corr.style.background_gradient(cmap='Greens', axis=0)
+
+
+##############################################################################
+# Now let's visualize a histogram of the price of the houses.
+
+
+fig, axs = plt.subplots(1, 1, figsize=(5, 5))
+axs.hist(y, bins=50)
+axs.set_xlabel("Median price of houses")
+axs.set_title("Histogram of house prices")
+axs.xaxis.set_major_formatter(FormatStrFormatter('%.0f'+"k"))
+plt.show()
+
+
+##############################################################################
+# Let's now create the different splits for the dataset, with a training,
+# calibration and test set.
+
+
+X_train, X_test, y_train, y_test = train_test_split(
+    X,
+    y['MedHouseVal'],
+    random_state=random_state
+)
+X_train, X_calib, y_train, y_calib = train_test_split(
+    X_train,
+    y_train,
+    random_state=random_state
+)
+
+
+##############################################################################
+# 2. Optimizing estimator
+# --------------------------------------------------------------------------
+# Optimization of the `LGBMRegressor` using `RandomizedSearchCV` to find the
+# optimal model to predict the house prices.
+
+
+estimator = LGBMRegressor(
+    objective='quantile',
+    alpha=0.5,
+    random_state=random_state
+)
+params_distributions = dict(
+    num_leaves=randint(low=10, high=50),
+    max_depth=randint(low=3, high=20),
+    n_estimators=randint(low=50, high=300),
+    learning_rate=uniform()
+)
+optim_model = RandomizedSearchCV(
+    estimator,
+    param_distributions=params_distributions,
+    n_jobs=-1,
+    n_iter=100,
+    cv=KFold(n_splits=5),
+    verbose=-1
+)
+optim_model.fit(X_train, y_train)
+estimator = estimator.set_params(**optim_model.best_params_)
+
+
+##############################################################################
+# 3. Comparaison of MAPIE methods
+# --------------------------------------------------------------------------
+# We will now proceed to compare the different regression method available
+# on MAPIE. For this tutorial we will compare the "naive", "jackknife plus ab",
+# "cv plus" and "conformalized quantile regression".
+
+
+def sort(
```

Comment:
As discussed before vacation, typing checks won't be performed on examples with PR #211. Also, you can remove the description of the parameters for sake of compactness.

Reply:
done